xarray/tests/test_dataarray.py

```diff
@@ -7313,3 +7326,31 @@ def test_deepcopy_obj_array():
     x0 = DataArray(np.array([object()]))
     x1 = deepcopy(x0)
     assert x0.values[0] is not x1.values[0]
+
+
+def test_clip(da):
```

Comment:
```suggestion
@pytest.mark.parametrize("backend", ["numpy", "dask"])
def test_clip(da):
```

to test dask?

Reply:
This already happens though?

Reply:
Oh..... I see. You've explicitly added it only for `rolling_exp` which can only accept numpy. Got it